app/models/slide_search.rb

```diff
@@ -0,0 +1,14 @@
+class SlideSearch
+  include ActiveModel::Model
+  attr_reader :search, :slides
+
+  def initialize(ransack_params)
+    # Hack to delete params can't be used by ransack
+    tag_search = ransack_params['tag_search'].split(',') if ransack_params.present? && ransack_params['tag_search'].present?
```

Comment:
__[Reek]__

[FeatureEnvy] SlideSearch#initialize refers to 'ransack_params' more than self (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->